extensions/typescript-language-features/src/ui/jsNodeWalkthrough.electron.ts

```diff
@@ -0,0 +1,196 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+import * as cp from 'child_process';
+
+import { Disposable } from '../utils/dispose';
+import { CommandManager } from '../commands/commandManager';
+
+const localize = nls.loadMessageBundle();
+
+export async function nodeWasResolvable(): Promise<boolean> {
```

Comment:
do all these functions need to be exported? it looks like only JsWalkthroughState and registerJsNodeWalkthrough are used externally